pkgs/applications/editors/sigil/default.nix

```
@@ -24,6 +24,10 @@ mkDerivation rec {
     python3Packages.lxml
   ];
 
+  prePatch = ''
+    sed -i '6d' src/Resource_Files/bash/sigil-sh_install
```

This is likely to silently break if the package is updated. Let's do something else, like

```suggestion
    sed -i '/^QTLIB_DIR=/ d' src/Resource_Files/bash/sigil-sh_install
```